docs/source/cupy-reference/difference.rst

```diff
@@ -0,0 +1,55 @@
+Difference between CuPy and NumPy
+=================================
+
+The interface of CuPy is designed to obey the interface of NumPy.
+However it has some differenece.
+
+
+Cast behavior from float to integer
+-----------------------------------
+
+Some casting behavior from float to integer is not defined.
+For example from a negative float to unsigned integer and infinity to integer.
+It depends on your CPU.
```

Comment:
NumPy developers *can* define its behavior and implement it with `if` statement, but they don't define its behavior and instead uses CPU behavior directly. For example, Java VM *defines* casting behaviors in Java VM specification.
It is main reason that we don't emulate CPU behavior with GPU in CuPy, I think.